pytorch_lightning/trainer/callback_config.py

```diff
@@ -32,79 +32,47 @@ def save_checkpoint(self, *args):
     def is_overridden(self, *args):
         """Warning: this is just empty shell for code implemented in other class."""
 
-    def configure_checkpoint_callback(self):
+    def configure_checkpoint_callback(self, checkpoint_callback):
         """
         Weight path set in this priority:
         Checkpoint_callback's path (if passed in).
         User provided weights_saved_path
         Otherwise use os.getcwd()
         """
-        ckpt_path = self.default_root_dir
-        if self.checkpoint_callback:
-            # init a default one
-            if self.logger is not None and self.logger.experiment is not None:
```

Comment:
without these lines, ddp experiments will not work correctly as they'll try to do the same thing multiple times.

Reply:
it is still there, @jeremyjordan only moved it to the checkpoint where it configures the paths. 
I also made sure that your recent change on master involving `and self.logger.experiment is not None:` is there too. 
See model_checkpoint.py